.travis.yml

```diff
@@ -223,8 +222,7 @@ jobs:
         - go run build/ci.go xcode -signer IOS_SIGNING_KEY -deploy trunk -upload gethstore/builds
 
     # This builder does the Azure archive purges to avoid accumulating junk
-    - stage: build
-      if: type = cron
```

Comment:
There was a `cron` limitation here that you nuked.

Reply:
Oops